### ISSUE TITLE
duniverse pull is too quiet

### ISSUE THREAD

**Issue by samoht** [ocamllabs/opam-monorepo#97]
It seems that `duniverse pull` is now a bit quieter ; it's nice but it's *quiet* now, eg.

```
duniverse pull
==> Using pull mode Source
[.. wait 5 minutes ..]
Successfully pulled 97/97 repositories
```
The `Successfully pulled 97/97 repositories` line should probably be printed more regularly on to show a bit of progress.

**Comment by avsm** [comment]
Yes -v will show more information but this needs to be a bit louder. It would be so nice to have a terminal progress bar library like Go has!

**Comment by NathanReb** [comment]
I know they had some progress bar code hanging around at Cryptosense, if they'd be willing to open source it, might be a good place to start!

CC @bbc2

**Comment by bbc2** [comment]
We indeed have some internal progress bar code but it would need some changes before being open-sourced, so I wouldn't count on that in the short-term.

More recently we've added a really simple progress bar to our API client: https://github.com/cryptosense/api-client/blob/297e7cde9a8c914eef1b1dd9de12df9581977235/multipart-form-writer/progress_bar.ml.  It doesn't have a lot of features but you might find it interesting.
